script.module.clouddrive.common/changelog.txt

```diff
@@ -0,0 +1,2 @@
+v1.0.0 - Nov 6, 2017:
+- First release
```

Comment:
FYI changelogs are no longer used, use the <news> addon.xml parameter.

Reply:
Ok. I'll move it to the news section.